sdks/python/apache_beam/utils/profiler.py

```diff
@@ -88,7 +88,11 @@ def __exit__(self, *args):
       self.profile_output = dump_location
 
     if self.log_results:
-      s = io.StringIO()
+      try:
+        import StringIO  # Python 2
```

Comment:
NIT: Do we want to add a comment something like `# TODO(BEAM-8371): Python2 should use StringIO` so it's easy to find it when we cleanup Py2 -> Py3?

Reply:
There are several keywords we'll have to look for during this cleanup. "Python 2" should be one of them, that's why I added it. Tracking all possible keywords to search in https://issues.apache.org/jira/browse/BEAM-7372.